src/main/java/com/klin/holoItems/utility/Utility.java

```diff
@@ -142,6 +142,75 @@ BlockFace.EAST, new Vector(1, 0, 0),
         put(DyeColor.GREEN, Stream.of(EMERALD_BLOCK/*, GREEN_BED, GREEN_CARPET*/, GREEN_CONCRETE/*, GREEN_CONCRETE_POWDER*/, GREEN_GLAZED_TERRACOTTA/*, GREEN_SHULKER_BOX*/, GREEN_STAINED_GLASS, GREEN_STAINED_GLASS_PANE, GREEN_TERRACOTTA, GREEN_WOOL, MOSS_BLOCK/*, MOSS_CARPET*/).collect(Collectors.toSet()));
         put(DyeColor.LIME, Stream.of(/*LIME_BED, LIME_CARPET,*/ LIME_CONCRETE/*, LIME_CONCRETE_POWDER*/, LIME_GLAZED_TERRACOTTA/*, LIME_SHULKER_BOX*/, LIME_STAINED_GLASS, LIME_STAINED_GLASS_PANE, LIME_TERRACOTTA, LIME_WOOL, SLIME_BLOCK).collect(Collectors.toSet()));
         put(DyeColor.BROWN, Stream.of(ANCIENT_DEBRIS/*, BROWN_BED, BROWN_CARPET*/, BROWN_CONCRETE/*, BROWN_CONCRETE_POWDER*/, BROWN_GLAZED_TERRACOTTA, BROWN_MUSHROOM_BLOCK/*, BROWN_SHULKER_BOX*/, BROWN_STAINED_GLASS, BROWN_STAINED_GLASS_PANE, BROWN_TERRACOTTA, BROWN_WOOL, COARSE_DIRT, DIRT, PODZOL, ROOTED_DIRT).collect(Collectors.toSet())); }};
+    /**
+     * Nutrition values from eating certain foods.
+     * Note: Chorus fruit and cake aren't listed here.
+     */
+    public static final Map<Material, float[]> nutritionValues = new HashMap<>(){{
```

Comment:
`EnumMap`

Reply:
It's possible to get these values from the server internals by using reflection.
Relevant packages are:
net.minecraft.world.food
net.minecraft.world.item

Reply:
"Convert to nms ItemStack" Ah, I was avoiding using NMS because I wasn't sure if we were allowed to have it, but since we are, this doesn't have to be hardcoded... might even be able to remove the variable entirely.

Reminder to self: Confirm what happens if you AutoEat Chorus Fruit.

Reply:
See discussion in discord

Reply:
We'd better move the Class and Method reflection stuff out to a new class so it'll be more maintainable and not such a mess. For iterating the Materials, it's gonna be:
```java
for (final Material material : Material.values()) {
    if (material.isEdible() && !material.isLegacy()) {
        ... // do the food property getting stuff and insert the values into the map
    }
}